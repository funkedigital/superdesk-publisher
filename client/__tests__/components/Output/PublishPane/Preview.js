import React from "react";
import Preview from "../../../../components/Output/PublishPane/Preview";
import { render, wait } from "@testing-library/react";
import Store from "../../../../components/Output/Store";

const item = {
  tenant: { domain_name: 'test.com' },
  route: { id: 1 }
}

describe("Output/PublishPane/Preview", () => {

  it("renders correctly", async () => {
    const { container, } = render(
      <Store.Provider
        value={{
          publisher: { getToken: () => '123' },
          selectedItem: { id: 1 }
        }}
      >
        <Preview
          isOpen={true}
          item={item}
          close={jest.fn()}
        />
      </Store.Provider>
    );





    expect(container.firstChild).toMatchSnapshot();
  });

});
