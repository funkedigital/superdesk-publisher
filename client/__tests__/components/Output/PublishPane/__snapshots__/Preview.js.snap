// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output/PublishPane/Preview renders correctly 1`] = `null`;
